omod/src/main/java/org/openmrs/module/appui/fragment/controller/HeaderFragmentController.java

```diff
@@ -43,7 +43,13 @@ public void controller(@SpringBean AppFrameworkService appFrameworkService, Frag
             fragmentModel.addAttribute("loginLocations", appFrameworkService.getLoginLocations());
 
             List<Extension> exts = appFrameworkService.getExtensionsForCurrentUser(AppUiExtensions.HEADER_CONFIG_EXTENSION);
-            Map<String, Object> configSettings = exts.size() > 0 ? exts.get(0).getExtensionParams() : null;
+            Extension lowestOrderExtension = exts.size() > 0 ? exts.get(0) : null;
```

Comment:
Do you think you can add some tests for this?

Reply:
Sure, I will do it